minet/cli/twitter/tweets.py

```diff
@@ -0,0 +1,71 @@
+import casanova
+from twitwi import (
+    normalize_tweet,
+    format_tweet_as_csv_row
+)
+from twitwi.constants import TWEET_FIELDS
+from twitter import TwitterHTTPError
+
+from ebbe import as_chunks
+
+from minet.cli.utils import LoadingBar
+from minet.twitter import TwitterAPIClient
+
+
+def twitter_tweets_hydration_action(cli_args):
+
+    client = TwitterAPIClient(
+        cli_args.access_token,
+        cli_args.access_token_secret,
+        cli_args.api_key,
+        cli_args.api_secret_key
+    )
+
+    enricher = casanova.enricher(
+        cli_args.file,
+        cli_args.output,
+        keep=cli_args.select,
+        add=TWEET_FIELDS,
+        total=cli_args.total
+    )
+
+    loading_bar = LoadingBar(
+        'Retrieving tweets metadata',
+        total=enricher.total,
+        unit='tweet_id'
+    )
+
+    for chunk in as_chunks(100, enricher.cells(cli_args.column, with_rows=True)):
+        tweets = ','.join(row[1] for row in chunk)
+        kwargs = {'_id': tweets}
```

Comment:
Here is where you should add the fields I told you in https://github.com/medialab/twitwi/issues/27